src/content/es/2019/media.md

```diff
@@ -0,0 +1,413 @@
+
+El test <i lang="en">[Lighthouse](./methodology#lighthouse)</i> es una comparación A/B con una encriptación progresiva del JPEG. Esto facilita una guía para saber qué imágenes pueden ser optimizadas un poco más con técnicas sin pérdida y potencialmente con técnicas con pérdida como usar diferentes niveles de calidad.
+
+ <figure>
+  <a href="/static/images/2019/media/fig12_percentage_optimized_images.png">
+    <img src="/static/images/2019/media/fig12_percentage_optimized_images.png" alt="Figura 12. Porcentaje de imágenes 'optimizadas'." aria-labelledby="fig12-caption" aria-describedby="fig12-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=1569150767">
+  </a>
+  <div id="fig12-description" class="visually-hidden">Gráfico de barras mostrando que en el percentil p10 el 100% de las imágenes están optimizadas, igual que en el percentil p25, en el percentil p50 el 98% de las imágenes están optimizadas (un 2% no lo están), en el percentil p75 un 83% de las imágenes están optimizadas (un 17% no lo están), y en el percentil p90 un 59% de las imágenes están optimizadas y un 41% no lo están.</div>
+  <figcaption id="fig12-caption">Figura 12. Porcentaje de imágenes 'optimizadas'</figcaption>
+</figure>
+
+La ventaja de este test AB <i lang="en">Lighthouse</i> no es solo la potencial reducción de bytes, la cual puede suponer bastantes MBs en el p95, sino que también muestra la mejora del rendimiento de la página.
+
+ <figure>
+  <a href="/static/images/2019/media/fig13_project_perf_improvements_image_optimization.png">
+    <img src="/static/images/2019/media/fig13_project_perf_improvements_image_optimization.png" alt="Figura 13. Estimación de la mejora del rendimiento de la página tras la optimización de imagen de Lighthouse." aria-labelledby="fig12-caption" aria-describedby="fig13-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=167590779">
+  </a>
+  <div id="fig13-description" class="visually-hidden">Gráfico de barras que muestra  que en el percentil p10 0 ms pudieron ser medidos, lo mismo pasa en el percentil p25, en el percentil p50 se redujeron 150 ms, en el percentil p75 se redujeron 1.460 ms, y en el percentil p90 se redujeron 5.720 ms.</div>
+  <figcaption id="fig13-caption">Figura 13. Estimación de la mejora del rendimiento de la página tras la optimización de imagen de <i lang="en">Lighthouse</i>.</figcaption>
+</figure>
+
+### Imágenes adaptables (<i lang="en">responsive</i>)
```

Comment:
We generally don't put the `<i lang="en"></i>` elements in headings as it messes up the automatically created short links. See discussion here: #656 

```suggestion
### Imágenes adaptables
```